src/universal-app/theme.scss

```diff
@@ -0,0 +1,16 @@
+@import '../lib/core/theming/all-theme';
```

Comment:
Couldn't we reuse one of the prebuilt themes here? That's what we do for the unit tests: https://github.com/angular/material2/blob/master/test/karma.conf.js#L39

Reply:
I could if you feel strongly that we should do that. I did it this way because I plan to work on some prototypes with MDC that would add optional flags to the theme mixins.

Reply:
I don't feel strongly about it, I wasn't aware that we needed it for more than just a theme.